Workbooks/WorkbooksMetadata.json

```diff
@@ -544,7 +544,14 @@
     "title": "Security Status",
     "templateRelativePath": "SecurityStatus.json",
     "subtitle": "",
-    "provider": "Microsoft"
+    "provider": "Microsoft",
+    "author": {
+      "name": "Microsoft"
+    },
+    "support": {
+      "tier": "Microsoft"
+    },
+    "categories": ["IT Operations", "Security - Others", "Compliance"]
```

Comment:
I saw the categories in workbook metadata schema like this:
categories{
   vertical: []
   domains: []
}
 Would it be better to update the categories here as well?

Reply:
thanks. will follow up with preeti

Reply:
fixed